docs/quickstarts/twitter.rst

```diff
@@ -84,3 +84,9 @@ you can use all the normal ``requests`` methods, like
 :meth:`~requests.Session.get` and :meth:`~requests.Session.post`,
 to make HTTP requests. If you only specify the path component of the URL,
 the domain will default to ``https://www.googleapis.com``.
+
+Frontend Demo
```

Comment:
This doesn't belong in this pull request. Adding a list of example projects should not mean modifying any of the quickstarts.